src/ontology/cl-edit.owl

```diff
@@ -32837,6 +32838,17 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1038/s41467-022-33324-
 AnnotationAssertion(rdfs:label obo:CL_4047030 "pre-venule capillary cell"@en)
 SubClassOf(obo:CL_4047030 obo:CL_0002144)
 
+# Class: obo:CL_4047032 (secretory pericyte)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.3389/fncel.2019.00282") obo:IAO_0000115 obo:CL_4047032 "A specialized pericyte located within the basement membrane of a blood capillary. This cell is characterized by its ability to produce and release a variety of bioactive molecules, collectively known as the pericyte secretome")
+AnnotationAssertion(terms:contributor obo:CL_4047032 <https://orcid.org/0009-0005-7919-4905>)
```

Comment:
Small suggested change: please include information from the axioms in the text

This cell is characterized by its ability to produce and release a variety of bioactive molecules, collectively known as the pericyte secretome.

--> 

This cell is characterized by its ability to produce and release a variety of bioactive molecules, collectively known as the pericyte secretome, including cytokines and regulators of angiogenesis.